### PR TITLE
use core SDK do CRUD and DSL create pipeline

### PR DESCRIPTION
**Pull request by cloga**
# PR into Azure/azureml-examples

## Checklist

I have:

- [x] read and followed the contributing guidelines

## Changes

- change examples to better align with: https://github.com/Azure/azureml_run_specification/tree/pipeline_sdk/specs/python-sdk/pipeline, which will use core SDK to do component and job CRUD, and use DSL to build pipeline

fixes #

- hidden local input dataset seems it didn't work now.
- add env version to iris standalone job.

### CODE REVIEW

File: sdk/jobs/single-step/scikit-learn/iris/iris-scikit-learn.ipynb [hunk 1]
```diff
@@ -42,6 +42,18 @@
...
   },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "#Enter details of your AML workspace\n",
+    "subscription_id = '15ae9cb6-95c1-483d-a0e3-b1a1a3b06324'\n",
+    "resource_group = 'user_studies'\n",
```
Comment (wangchao1230) [inline sdk/jobs/single-step/scikit-learn/iris/iris-scikit-learn.ipynb:53]: update this

Comment (cloga) [reply sdk/jobs/single-step/scikit-learn/iris/iris-scikit-learn.ipynb]: removed.

**Comment by luigiw** [comment]
Thx for the update. Have you tested that `dsl.load_component` is working as of today?

**Comment by cloga** [comment]
Yes, I have tested these samples, `dsl.load_component` works today.

**Pull request closed without merging**